openex-front/src/admin/components/teams/players/PlayerForm.tsx

```diff
@@ -43,7 +43,15 @@ const PlayerForm: FunctionComponent<PlayerFormProps> = ({
       .string()
       .regex(
         /^\+/,
-        t("Invalid input. Please use '+' character and country identifier."),
+        t('Invalid input. Please use \'+\' character and country identifier.'),
+      )
+      .optional()
+      .nullable(),
+    user_phone2: z
+      .string()
+      .regex(
+        /^\+/,
+        t('Invalid input. Please use \'+\' character and country identifier.'),
```

Comment:
Can you change the UpdateUserInput & UpdatePlayerInput class and add this annotation 
  @Pattern(regexp="^\\+[\\d\\s]*", message = PHONE_FORMAT) ?

Exemple :
  @JsonProperty("user_phone")
  @Pattern(regexp="^\\+[\\d\\s]*", message = PHONE_FORMAT)
  private String phone;